manifests/cyclone.yaml

```diff
@@ -39,6 +39,7 @@ data:
       "pvc": "",
       "secret": "",
       "cyclone_server_addr": "native-cyclone-server.default.svc.cluster.local"
```

Comment:
Need to add `,` at end of the line.

`native-cyclone-server.default` is enough for `cyclone_server_addr`.